apps/qubit/modules/menu/actions/quickLinksMenuComponent.class.php

```diff
@@ -28,12 +28,26 @@ class menuQuickLinksMenuComponent extends sfComponent
 {
   public function execute($request)
   {
-    // Get menu objects
-    $this->quickLinksMenu = QubitMenu::getById(QubitMenu::QUICK_LINKS_ID);
+    // Get menu
+    $quickLinksMenu = QubitMenu::getById(QubitMenu::QUICK_LINKS_ID);
 
-    if (!$this->quickLinksMenu instanceof QubitMenu)
+    if (!$quickLinksMenu instanceof QubitMenu)
     {
       return sfView::NONE;
     }
+
+    // Get menu items that correspond to an external URL or an internal path
+    $this->quickLinks = [];
+
+    foreach ($quickLinksMenu->getChildren() as $child)
+    {
+      $url = $child->getPath(['getUrl' => true, 'resolveAlias' => true]);
+      $urlParsed = parse_url($url);
+
+      if (isset($urlParsed['scheme']) || QubitObject::actionExistsForUrl($url))
```

Comment:
There's a bug in this logic somewhere - `$this->quickLinks` is always empty in my test AtoM instance, even though I have quick link menu items.

![empty_quick_links_menu](https://user-images.githubusercontent.com/626135/101530821-40f21400-3947-11eb-89ef-9a0a43bde2f0.png)

Reply:
I retested, and the quick links are working for me now.